stories/Molecules/Blocks/StatsCards/statscards.scss

```diff
@@ -0,0 +1,217 @@
+@import '../../../assets/scss/breakpoints';
+@import '../../../assets/scss/variables';
+
+/* statscard start */
+.statscard {
+  background-color: $color-gray-200;
+
+  & span {
+    display: block;
+  }
+
+  &__title {
+    color: $color-black;
+    font-family: $Font-Family-ProximaBold;
+    letter-spacing: 0;
+    margin-bottom: $spacing-40;
+
+    @include devicebreak(medium) {
+      margin-bottom: $spacing-80;
+    }
+  }
+
+  &__number {
+    color: $color-gray-200;
+    font-family: $Font-Family-SohneBold;
+    font-size: 80px;
+    letter-spacing: 0.8px;
+    line-height: 87.2px;
+    margin-bottom: $spacing-24;
+    text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
+    -webkit-text-stroke: 2px $color-black;
```

Comment:
@Shubham-Sharma-77 is this property supported on all the browsers

Reply:
@surbhi90 -WebKit-text are not support in IE, so we can use text-shadow here